test/acceptance/features/environment.py

```diff
@@ -14,12 +14,19 @@
 
 from steps.command import Command
 from steps.environment import ctx
+from behave.model_core import Status
 
 import os
 import semver
 
 cmd = Command()
 
+if os.getenv("DELETE_NAMESPACE") in ["alwaysdelete", "neverdelete", "keepwhenfailed"]:
+    delete_namespace = os.getenv("DELETE_NAMESPACE")
+else:
+    delete_namespace = "alwaysdelete"
+scenario_status = True
```

Comment:
Instead of using a plain global variable, please use the `Environment` class (`test/acceptance/features/steps/environment.py`) and add it there... you could then access it via global `ctx` similarly as `ctx.cli`

Reply:
Or better `no_scenarios_failed` which is more accurate